src/main/listeners.ts

```diff
@@ -59,10 +61,18 @@ export function setupListeners() {
     return { path: desktopAppPath, exists: exists };
   });
 
-  ipc_deleteFolder.main!.handle(async ({ path }) => {
-    await fs.remove(path).catch((err) => {
-      throw new Error(err);
-    });
+  ipc_migrateDolphin.main!.handle(async ({ migrateNetplay, migratePlayback, desktopAppPath }) => {
+    if (migrateNetplay) {
+      await dolphinManager.copyDolphinConfig(DolphinLaunchType.NETPLAY, migrateNetplay);
+    }
+    if (migratePlayback) {
+      await dolphinManager.copyDolphinConfig(DolphinLaunchType.PLAYBACK, migratePlayback);
+    }
+    if (desktopAppPath) {
+      await fs.remove(desktopAppPath).catch((err) => {
+        throw new Error(err);
+      });
```

Comment:
```suggestion
      await fs.remove(desktopAppPath);
```
There's no point catching then throwing the same error.